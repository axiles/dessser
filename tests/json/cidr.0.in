{ "v4": { "ip": 1234, "mask": 24 } }
