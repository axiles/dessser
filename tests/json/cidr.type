//[ v4 { ip: u32; mask: u8 } | v6 { ip : u128; mask: u8 } ]
cidr
