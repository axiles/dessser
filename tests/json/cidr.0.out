(0 (1234 24))
